README.md

```diff
@@ -207,16 +207,34 @@ This project is licensed under the MIT License - see the [LICENSE](LICENSE) file
 ## API
 Account management exposes various endpoints that aid in managing accounts within the P&I service.
 
-- POST `/account/add/pi` - used to add a user to the P&I user database. Takes in a header of the email of the admin
-issuing
+`/account/add/azure` - used to add an account into the azure AAD. Takes in users name, email and role, and attempts to
```

Comment:
The REST method prefix needs adding in here (and for the one below)

Reply:
^^

Reply:
sorted :)